tsc error when trying to build using the getting started instructions

# Description

I followed the getting started instructions from the master branch README in a new project. When running tsc on my project, I received the following:

```
$ tsc
node_modules/express-zod-api/dist/index.d.ts:1:25 - error TS2307: Cannot find module 'compression' or its corresponding type declarations.

1 import compression from 'compression';
                          ~~~~~~~~~~~~~

node_modules/express-zod-api/dist/index.d.ts:3:37 - error TS2307: Cannot find module 'express-fileupload' or its corresponding type declarations.

3 import * as express_fileupload from 'express-fileupload';
                                      ~~~~~~~~~~~~~~~~~~~~

node_modules/express-zod-api/dist/index.d.ts:4:41 - error TS2307: Cannot find module 'express-fileupload' or its corresponding type declarations.

4 import express_fileupload__default from 'express-fileupload';
                                          ~~~~~~~~~~~~~~~~~~~~

node_modules/express-zod-api/dist/index.d.ts:8:21 - error TS2307: Cannot find module 'winston' or its corresponding type declarations.

8 import Winston from 'winston';
                      ~~~~~~~~~

node_modules/express-zod-api/dist/index.d.ts:523:40 - error TS2344: Type '"body" | "headers" | "query" | "params" | "files"' does not satisfy the constraint 'keyof Request<ParamsDictionary, any, any, ParsedQs, Record<string, any>>'.
  Type '"files"' is not assignable to type 'keyof Request<ParamsDictionary, any, any, ParsedQs, Record<string, any>>'.

523 type InputSource = keyof Pick<Request, "query" | "body" | "files" | "params" | "headers">;

```

## Expected

I expected the getting started instructions to result in a successful tsc build.

## Reproduction

Copy and paste the code snippets in the getting started section of the README.

## Context

- Node.js version: 20.9.0
- `express-zod-api` version: 17.7.0

Also worth noting that i'm attempting to pass an instance of Pino rather than letting express-zod-api instantiate a winston logger for me.

--- a/README.md
+++ b/README.md
@@ -129,12 +129,13 @@ npm install express-zod-api express zod winston typescript http-errors
 npm install -D @types/express @types/node @types/http-errors
 ```
 
-Add the following option to your `tsconfig.json` file in order to make it work as expected:
+Ensure having following options in your `tsconfig.json` file in order to make it work as expected:
 
 ```json
 {
   "compilerOptions": {
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true,
   }
 }
 ```

--- a/README.md
+++ b/README.md
@@ -135,7 +135,7 @@ Ensure having following options in your `tsconfig.json` file in order to make it
 {
   "compilerOptions": {
     "strict": true,
-    "skipLibCheck": true,
+    "skipLibCheck": true
   }
 }
 ```

--- a/README.md
+++ b/README.md
@@ -129,7 +129,7 @@ npm install express-zod-api express zod winston typescript http-errors
 npm install -D @types/express @types/node @types/http-errors
 ```
 
-Ensure having following options in your `tsconfig.json` file in order to make it work as expected:
+Ensure having the following options in your `tsconfig.json` file in order to make it work as expected:
 
 ```json
 {

